Разметка главной страницы

Regeace opened the pull request:
Добавляет разметку в index.html, добавляет файл каталога отелей hotel-catalogue.html (в нём на данный момент только предварительная разметка хедера и футера).

---
:mortar_board: [Разметка главной страницы](https://up.htmlacademy.ru/htmlcss/33/user/1812401/tasks/3)

:boom: https://htmlacademy-htmlcss.github.io/1812401-sedona-33/2/

Regeace committed:
Добавляет разметку в index

Базовая разметка в index.html, а также добавлен файл hotel-catalogue.html с разметкой хедера и футера страницы.

Regeace committed:
Улучшает разметку

Орфографические правки, изменения в классах тегов.

LoremLipsum commented on index.html:
```diff
@@ -2,11 +2,149 @@
+    <header class="page-header">
+      <nav class="navigation">
+        <a class="navigation-logo" href="">
+          <img class="page-header-logo" src="" alt="Логотип городка Седона">
```
Внутри блока, есть элементы блока. Вот как здесь ссылка с картинкой. Ссылка - это блок лого, а картинка внутренний элемент. Класс здесь должен именоваться по названию блока `navigation-logo`,  для картинки это `navigation-logo-image`. Либо убери класс у картинки, и стилизуй по селектору `.navigation-logo img`
Сейчас у ссылки с лого и картинки лого, разные классы, не связанные между собой и догадаться, читая только стили, что откуда, невозможно.

LoremLipsum commented on index.html:
```diff
@@ -2,11 +2,149 @@
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS. Профессиональная вёрстка сайтов</a>».</p>
+        <ul class="navigation-list">
+          <li class="navigation-item">
+            <a class="navigation-link navigation-link-current">Главная</a>
```
Давай сделаем более[ доступный вариант оформления для текущей страницы](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current), используй не класс (его убери), и добавь ариа-атрибут `aria-current="page"` href тоже верни

LoremLipsum commented on index.html:
```diff
@@ -2,11 +2,149 @@
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">
+              <span class="visually-hidden">Мне нравится</span>
```
Можно ещё написать Избранное

LoremLipsum commented on index.html:
```diff
@@ -2,11 +2,149 @@
+            </a>
+          </li>
+          <li class="navigation-item">
+            <span class="visually-hidden">Забронировать отель</span>
```
Это лишнее, у кнопки уже есть текст. Ну хотя можно оставить как дополнительное текстовое описание. На твоё усмотрение

LoremLipsum commented on index.html:
```diff
@@ -2,11 +2,149 @@
+          </li>
+          <li class="navigation-item">
+            <span class="visually-hidden">Забронировать отель</span>
+            <button class="button button-brown">Хочу сюда!</button>
```
Это ссылка

LoremLipsum commented on index.html:
```diff
@@ -2,11 +2,149 @@
+    
+    <main class="main-index">
+      <h1 class="visually-hidden">Седона</h1>
+      <img class="mainpaige-logo" src="" alt="Логотип городка Седона">
```
Картинку надо обернуть в див или секшион, это заходная часть или интро страницы, если обернешь в секшион, то нужно добавить скрытый заголовок. Див или секшион, выбери исходя от того какая структура тебе больше понравится. Может ты хочешь чтобы все главные блоки на странице были обернуты в секшион. Если обернёшь в див, для читалки это будет просто картинка. Или ещё есть вариант обернуть интро в хэдер, то есть в отдельный хэдер который будет находится в мейне, а не главный хэдер. И на странице каталога, тоже для заходной картинки сделай как здесь, чтобы было одинаково. Но классы напиши им разные, они достаточно различаются, и лучше написать отдельные стили для интро на главной и внутренней странице

LoremLipsum commented on index.html:
```diff
@@ -2,11 +2,149 @@
+      <img class="mainpaige-logo" src="" alt="Логотип городка Седона">
+      
+      <section class="advantages">
+        <b class="slogan">Седона — небольшой городок в Аризоне,<br>заслуживающий большего!</b>
```
Заголовок

LoremLipsum commented on index.html:
```diff
@@ -2,11 +2,149 @@
+      
+      <section class="advantages">
+        <b class="slogan">Седона — небольшой городок в Аризоне,<br>заслуживающий большего!</b>
+        <h2 class="visually-hidden">Преимущества</h2>
```
Это лишнее

LoremLipsum commented on index.html:
```diff
@@ -2,11 +2,149 @@
+      <section class="advantages">
+        <b class="slogan">Седона — небольшой городок в Аризоне,<br>заслуживающий большего!</b>
+        <h2 class="visually-hidden">Преимущества</h2>
+        <p class="reasons-to-buy">Рассмотрим 5 причин, по которым Седона круче, чем Гранд-Каньон!</p>
```
класс reasons-text, не надо в названии класса описывать о чем текст

LoremLipsum commented on index.html:
```diff
@@ -2,11 +2,149 @@
+      <h1 class="visually-hidden">Седона</h1>
+      <img class="mainpaige-logo" src="" alt="Логотип городка Седона">
+      
+      <section class="advantages">
```
Почему здесь класс преимущества, это же причины. Старайся именовать так, чтобы можно было читать ситли, не глядя в разметку.

LoremLipsum commented on index.html:
```diff
@@ -2,11 +2,149 @@
+        <b class="slogan">Седона — небольшой городок в Аризоне,<br>заслуживающий большего!</b>
+        <h2 class="visually-hidden">Преимущества</h2>
+        <p class="reasons-to-buy">Рассмотрим 5 причин, по которым Седона круче, чем Гранд-Каньон!</p>
+        <ul class="advantages-list">
```
Это список с причинами reasons-list. 

У нас пять причин, первая, вторая итд., идут по порядку. Значит и список надо упорядочить)

LoremLipsum commented on index.html:
```diff
@@ -2,11 +2,149 @@
+        <h2 class="visually-hidden">Преимущества</h2>
+        <p class="reasons-to-buy">Рассмотрим 5 причин, по которым Седона круче, чем Гранд-Каньон!</p>
+        <ul class="advantages-list">
+          <li class="advantages-item">
```
reasons-item

LoremLipsum commented on index.html:
```diff
@@ -2,11 +2,149 @@
+            <p class="advantages-item-number">— №1 —</p>
+            <p class="advantages-description">Седона — не аттракцион<br>для туристов, там течёт<br>своя жизнь</p>
+          </li>
+          <li class="advantages-item">
```
Не надо выносить картинку в отдельную лишку. Одна причина, одна лишка

LoremLipsum commented on index.html:
```diff
@@ -2,11 +2,149 @@
+        <ul class="advantages-list">
+          <li class="advantages-item">
+            <h3 class="advantages-title">Настоящий<br>городок</h3>
+            <p class="advantages-item-number">— №1 —</p>
```
Убери из разметки номера причин, сделаем [css-счётчик,](https://developer.mozilla.org/ru/docs/Web/CSS/CSS_Counter_Styles/Using_CSS_counters) используя псевдоэлемент для заголовков.

LoremLipsum commented on index.html:
```diff
@@ -2,11 +2,149 @@
+          <li class="advantages-item">
+            <img class="advantages-item-picture" src="" alt="Салун">
+          </li>
+          <li class="advantages-item">
```
Вот это список с преимуществами, оформи как неупорядоченный вложденный список внутри первой лишки, где первая причина. Это отдельный список с преимуществами, не надо их ложить в одну кучу

LoremLipsum commented on index.html:
```diff
@@ -2,11 +2,149 @@
+            <h3 class="advantages-title">Жильё</h3>
+            <p class="advantages-description">Рекомендуем пожить<br>в настоящем мотеле,<br>всё как в кино!</p>
+          </li>
+          <li class="advantages-item">
```
К остальнымпунктам такие же правки. Класс надо поменять на reasons. Картинки, это иллюстрация а не отдельная причина и должны быть внутри соответстующего пункта с причиной. Номера надо убрать из разметки

LoremLipsum commented on index.html:
```diff
@@ -2,11 +2,149 @@
+      </section>
+
+      <section class="hotel-search">
+        <h2 class="visually-hidden">Поиск отеля</h2>
```
лишнее, убери

LoremLipsum commented on index.html:
```diff
@@ -2,11 +2,149 @@
+
+      <section class="hotel-search">
+        <h2 class="visually-hidden">Поиск отеля</h2>
+        <b class="hotel-search-question">Заинтересовались?</b>
```
Заголовок

LoremLipsum commented on index.html:
```diff
@@ -2,11 +2,149 @@
+
+    <footer class="index-footer">
+      <section class="newsletter">
+        <h2 class="visually-hidden">Подписка на рассылку</h2>
```
убери

LoremLipsum commented on index.html:
```diff
@@ -2,11 +2,149 @@
+    <footer class="index-footer">
+      <section class="newsletter">
+        <h2 class="visually-hidden">Подписка на рассылку</h2>
+        <b class="newsletter-subscribe-ask">Подпишитесь на рассылку</b>
```
Сделай заголовком

LoremLipsum commented on index.html:
```diff
@@ -2,11 +2,149 @@
+        <b class="newsletter-subscribe-ask">Подпишитесь на рассылку</b>
+        <p class="newsletter-decription">Только полезная информация и никакого спама,<br>честное бойскаутское!</p>
+        <form class="newsletter-form" action="https://echo.htmlacademy.ru/" method="post">
+          <input class="field" name="newsletter-email" placeholder="Ваш e-mail" type="email" required>
```
Добавь label к полю

LoremLipsum commented on index.html:
```diff
@@ -2,11 +2,149 @@
+        </form>
+      </section>
+
+      <section class="footer-contacts">
```
Секцию можно назвать проще, просто social и убрать footer из классов. Это по желанию

Regeace committed:
Исправляет разметку

Вносит изменения в разметку главной страницы, а также добавляет предварительную разметку страницы-каталога.

Regeace committed:
Исправляет описание изображений

Атрибут alt изображений исправлен для соответствия критериям качества.

Regeace committed:
Улучшает разметку

Исправлена и доработана разметка. В основном: классы элементов.
